### ISSUE TITLE
Custom BlockList

### ISSUE THREAD

**Issue by PeterKW** [ChrisTitusTech/win10script#159]
https://github.com/Sycnex/Windows10Debloater 
Would be interesting to have a customisable list version of the Essential Tweaks so we can save which tweaks we want to use easily in the GUI

![image](https://user-images.githubusercontent.com/13779157/127707435-21abdc79-9829-44f0-b7be-2a2f73179352.png)

**Comment by ChrisTitusTech** [comment]
Yeah, I agree. If I can find a slick way to integrate this, I'll do it in a heartbeat. However, I don't like a lot of PowerShells list / checkbox functionality as it can be confusing.

**Comment by PeterKW** [comment]
> Yeah, I agree. If I can find a slick way to integrate this, I'll do it in a heartbeat. However, I don't like a lot of PowerShells list / checkbox functionality as it can be confusing. 

Thank you very much @ChrisTitusTech 
I have been merging your commits with a fork with some of the tweaks commented out and the option to install brave browser nightly build option textbox added. I've also been using the HardenTools script you recommended for Windows security and Winaero Twеaker with Windows 11.
